Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs

```diff
@@ -131,40 +130,8 @@ public void SubscribeAuth()
             Log.Trace("BitfinexBrokerage.SubscribeAuth(): Sent authentication request.");
         }
 
-        /// <summary>
-        /// Subscribes to the requested symbols (using an individual streaming channel)
-        /// </summary>
-        /// <param name="symbols">The list of symbols to subscribe</param>
-        public override void Subscribe(IEnumerable<Symbol> symbols)
-        {
-            foreach (var symbol in symbols)
-            {
-                if (_subscriptionManager.IsSubscribed(symbol) ||
-                    symbol.Value.Contains("UNIVERSE") ||
-                    !_symbolMapper.IsKnownLeanSymbol(symbol) ||
-                    symbol.SecurityType != _symbolMapper.GetLeanSecurityType(symbol.Value))
-                {
-                    continue;
-                }
-
-                _subscriptionManager.Subscribe(symbol);
-
-                Log.Trace($"BitfinexBrokerage.Subscribe(): Sent subscribe for {symbol.Value}.");
-            }
-        }
-
-        /// <summary>
-        /// Ends current subscriptions
-        /// </summary>
-        private void Unsubscribe(IEnumerable<Symbol> symbols)
-        {
-            foreach (var symbol in symbols)
-            {
-                _subscriptionManager.Unsubscribe(symbol);
 
-                Log.Trace($"BitfinexBrokerage.Unsubscribe(): Sent unsubscribe for {symbol.Value}.");
-            }
-        }
+        public override void Subscribe(IEnumerable<Symbol> symbols) { }
```

Comment:
We should add a comment of why this isn't required, and wasn't used in master either, same for `GetSubscribed`

Reply:
done